lib/msal-browser/docs/signed-http-request.md

```diff
@@ -0,0 +1,70 @@
+# Signed Http Request
+
+MSAL.js provides the `SignedHttpRequest` class as a convenience to assist in creating [signed http requests (SHRs)](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-signed-http-request-03) for payloads (e.g. tokens) that are acquired out-of-band from MSAL.js. This enables applications to leverage the same encryption and caching MSAL uses for [access token proof-of-possession](./access-token-proof-of-possession.md) for their own payloads.
+
+Note: The `SignedHttpRequest` class is intended for advanced use cases where the built-in access token proof-of-possession functionality cannot be used.
+
+## Requirements
+
+To use the `SignedHttpRequest` API, an application must:
+
+1. Forward the public key thumbprint returned to the server issuing the payload, and that server must embedded the thumbprint inside the payload. This is typically a signed JWT.
+2. Provide the payload and the original public key thumbprint back to MSAL.
+3. The resource server for which the SHR is intended must understand how to decode and validate the SHR and inner payload.
+
+
+### AAD Parameters
+
+For tokens issued by AAD, applications will need to include additional query parameters in their token requests:
```

Comment:
Unsupported or included by default for MSA?

Reply:
Sorry, this should say "AAD/MSA"